src/components/pages/brands-page/brands-page.ts

```diff
@@ -0,0 +1,126 @@
+import { TopMenu } from '@/components/top-menu/top-menu';
+import { Container } from '@/components/container/container';
+import { BaseComponent } from '@/templates/base-component';
+import './brands-page.scss';
+
+export class BrandsPage extends BaseComponent {
+  private container: Container;
+  private topMenu: TopMenu = new TopMenu();
+
+  constructor(private props: Record<string, string>) {
+    super('div', { className: 'brands' });
+
+    this.append(this.topMenu);
+
+    this.container = new Container('brands__container');
+    this.append(this.container);
+
+    const brandLetterB = new BaseComponent('div', { className: 'brands__letter' });
+    brandLetterB.setContent('B');
+    this.container.append(brandLetterB);
+
+    const brandsHolderB = new BaseComponent('div', { className: 'brands__holder' });
+    this.container.append(brandsHolderB);
```

Comment:
What if we would have more items? Create an array with objects for such cases and those objects would contain classname, content, etc. for each item.

```
const brands = [...];
brands.map(brand => {
    const brandLetter = new BaseComponent('div', { className: 'brands__letter' });
    brandLetter.setContent(brand.content);
    this.container.append(brandLetter);
    const brandsHolder = new BaseComponent('div', { className: 'brands__holder' });
    this.container.append(brandsHolder);
});
```